### DISCUSSION TITLE
How to add plug to HTW?

### DISCUSSION THREAD

**Discussion by zer0deck** [milo526/homebridge-tuya-web#211] [Q&A]
Maybe this kinda stupid q, btw i really dont understand how to do this. I have some tuya plugs, but no1 of them can be added to my homebridge. 
I've already tried to do this by any device type in "device settings", or hidden device or confi itself. I also have pir sensors, which also i cannot add. 
I'll be grateful for any answers, even "youre brain is like a nut, you should click this one button and everything will start to work.." 
This type of answers i find the best)

**Comment by milo526** [comment]
Please post the output of the API as explained in [the README under "How to check whether the API that this library uses can control your device"](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device). Without this it is impossible to say anything useful about your specific situation.
